default/buildings.txt

```diff
@@ -999,6 +1026,7 @@ BuildingType
             Planet environment = Adequate
         ]
     ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
```

Comment:
This is one of the buildings that people may rightly want to build more than one of; if multiple terraforms are desired the current structure allows time to be saved by having that be *mostly* (but not entirely) done in parallel (and there is no cost savings, it still costs the same as doing it completely sequentially).  Changing that to prevent any parallel construction could be a reasonable decision, but I would vote to not so restrict it, and at least it should not be changed without debate.